### DISCUSSION TITLE
Color pallete

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#170] [Developer Feedback]
There is a [page](https://docs.zepp.com/docs/designs/visual/color/) related to recommended pallete, but it's not very helpful for mere developer, as not everyone have dedicated designer. Additionally, some colors defined with a mask, but controls allow to define only color without opacity masking.
In some of examples for 2.x there is a constants file with colors, but it's incomplete.
To provide more consistent user experience with 3-rd party apps imho it would be helpful to have single source of truth. 

In attachment my approach to assemble defined colors
[testcolors.d.ts.md](https://github.com/zepp-health/zeppos-forum/files/11476674/testcolors.d.ts.md)

**Comment by MyoungXUE** [comment]
Thank you for your feedback. We will make adjustments based on your suggestions.
